src/include/duckdb/common/types/value.hpp

```diff
@@ -12,14 +12,23 @@
 #include "duckdb/common/exception.hpp"
 #include "duckdb/common/types.hpp"
 
+
+#ifdef _WIN32
+#define DUCKDBAPI __declspec(dllexport)
```

Comment:
Can this be moved to a common header (e.g. `duckdb/common/winapi.hpp`, which can then be included from `duckdb/common/helper.hpp` and `duckdb.h`)?